modules/apps/object/object-test/src/testFunctional/macros/ObjectNotifications.macro

```diff
@@ -3,57 +3,99 @@ definition {
 	macro addNewNotificationTemplate {
 		LexiconEntry.gotoAdd();
 
-		MenuItem.click(menuItem = ${notificationType});
+		if (${notificationType} == "Email") {
+			MenuItem.click(menuItem = ${notificationType});
```

Comment:
Instead of creating a mega macro to cover both, we can just split this into two separate macros. It makes them easier to read and understand